main.go

```diff
@@ -33,6 +34,9 @@ func createServer() {
 	// completedTasks
 	router.HandleFunc("/api/v1/completedTasks", completed_tasks_router.CreateCompletedTasks).Methods("POST", "OPTIONS")
 
+	// flowers
+	router.HandleFunc("/api/v1/flowers", flowers_router.GetFlowerList).Methods("GET", "OPTIONS")
```

Comment:
@yukari-n-erb I don't think we need the OPTIONS method for GET calls. So far it's only with POSTs, but PUTs and DELETEs may need it so we'll need to test this.